packages/aws-cdk-lib/aws-cloudfront/lib/web-distribution.ts

```diff
@@ -1118,6 +1134,9 @@ export class CloudFrontWebDistribution extends cdk.Resource implements IDistribu
 
     let s3OriginConfig: CfnDistribution.S3OriginConfigProperty | undefined;
     if (originConfig.s3OriginSource) {
+      if (originConfig.s3OriginSource.originAccessIdentity && originConfig.s3OriginSource.originAccessControl) {
+        throw Error('Only one of origin access identity or origin access control can be defined.');
+      }
```

Comment:
Would this block users from doing 2-step migration approach where they would have both OAI and OAC?

Reply:
Removed this as we are deprecating `CloudFrontWebDistribution` - was an old change I accidentally pushed.